src/main/java/org/broadinstitute/consent/http/resources/MetricsResource.java

```diff
@@ -24,19 +24,22 @@ public MetricsResource(MetricsService metricsService) {
   @Path("/dar/decision")
   @Produces(MediaType.TEXT_PLAIN)
   public Response getDarMetricsData() {
-    String joiner = "\t";
-    StringBuilder tsv = new StringBuilder(DarDecisionMetrics.getHeaderRow(joiner));
-    metricsService.generateDarDecisionMetrics().forEach(m -> tsv.append(m.toString(joiner)));
-    return Response.ok(tsv.toString()).build();
+    Type type = Type.DAR;
+    return getMetricsData(type);
   }
 
   @GET
   @Path("/dac/decision")
   @Produces(MediaType.TEXT_PLAIN)
   public Response getDacMetricsData() {
-    String joiner = "\t";
-    StringBuilder tsv = new StringBuilder(DacDecisionMetrics.getHeaderRow(joiner));
-    metricsService.generateDacDecisionMetrics().forEach(m -> tsv.append(m.toString(joiner)));
+    Type type = Type.DAC;
+    return getMetricsData(type);
```

Comment:
You could inline this
```suggestion
    return getMetricsData(Type.DAC);
```